modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -162,10 +162,10 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
             }
             {{/isListContainer}}
     {{/formParams}}{{/vendorExtensions}}
-    {{#hasFormParams}}{{#multipartFormData}}
+    {{#vendorExtensions}}{{#hasFormParams}}{{^multipartFormData}}
```

Comment:
how does this solve issue https://github.com/OpenAPITools/openapi-generator/pull/2695?